Description: temporariliy update "bashbrew-buildkit-env-setup" to support arches again (and to use a hand-compiled image for buildkit-in-Docker)

diff --git a/.bin/bashbrew-buildkit-env-setup.sh b/.bin/bashbrew-buildkit-env-setup.sh
index 190fb4740..ef0f82f24 100755
--- a/.bin/bashbrew-buildkit-env-setup.sh
+++ b/.bin/bashbrew-buildkit-env-setup.sh
@@ -52,8 +52,8 @@ _bashbrew_buildkit_env_setup() {
 	vars="$(_jq_setenv <<<"$vars" BASHBREW_BUILDKIT_SYNTAX "$dockerfileTag")"
 
 	case "${BASHBREW_ARCH:-}" in
-		nope) # amd64 | arm64v8) # TODO re-enable this once we figure out how to handle "docker build --tag X" + "FROM X" correctly all-local
-			BASHBREW_BUILDKIT_IMAGE="$(grep <<<"$externalPins" -m1 '^moby/buildkit:')"
+		*)
+			BASHBREW_BUILDKIT_IMAGE="$(grep <<<"$externalPins" -m1 '^tianon/buildkit:')" # TODO moby/buildkit (needs more arches)
 			BASHBREW_BUILDKIT_IMAGE="$(_resolve_external_pins "$BASHBREW_BUILDKIT_IMAGE")"
 			export BASHBREW_BUILDKIT_IMAGE
 
diff --git a/.external-pins/tianon/buildkit___0.12 b/.external-pins/tianon/buildkit___0.12
new file mode 100644
index 000000000..7595eccb3
--- /dev/null
+++ b/.external-pins/tianon/buildkit___0.12
@@ -0,0 +1 @@
+sha256:7b3523bb466d58423d2c3e0055b47c3f892d709da91fbd1eecbc0d2521154deb
